src/modules/dashboard/components/HookedTokenIcon/HookedTokenIcon.tsx

```diff
@@ -60,12 +71,22 @@ const HookedTokenIcon = ({
 
   useEffect(() => {
     const loadTokenLogo = async () => {
+      const image = localStorage.getItem(address);
+      if (image) {
+        setTokenImage(image);
+        return;
+      }
+
       if (!dontFetch && address && !iconName) {
         const response = await loadTokenImages(address);
-        if (!response.ok) {
-          return;
+        if (response.ok) {
+          const blob = await response.blob();
+          const base64image = await getBase64image(blob);
```

Comment:
As I've said above the `getBase64image` function doesn't need to be async

Reply:
I was testing no async way, and i was getting empty string

Reply:
I'll have  a look again